recipes/freebayes/meta.yaml

```diff
@@ -24,16 +25,16 @@ requirements:
     - htslib
     - parallel
     - meson
+    - python
```

Comment:
I don't think it is needed as host requirement.

Reply:
I believe this could be removed.  Will try.